tests/chainer_tests/test_function_node.py

```diff
@@ -960,14 +970,29 @@ def check_grad(self):
             self._print_variables('gxs (expected)', expected)
             raise
 
+    def chainerx_grad(self, device):
+        if self.loss_scale:
+            pytest.skip('Unsupported configurations')
```

Comment:
Let's be more specific.
```suggestion
            pytest.skip('chainerx.grad does not support loss_scale')
```